Makefile

```diff
@@ -9,8 +9,8 @@ export IMAGE_VERSION = v1.33.0
 export SIMPLE_VERSION := $(shell (test "$(shell git describe --tags)" = "$(shell git describe --tags --abbrev=0)" && echo $(shell git describe --tags)) || echo $(shell git describe --tags --abbrev=0)+git)
 export GIT_VERSION := $(shell git describe --dirty --tags --always)
 export GIT_COMMIT := $(shell git rev-parse HEAD)
-export K8S_VERSION = 1.27.0
-export ENVTEST_VERSION = 1.27.x
+export K8S_VERSION = 1.28.0
```

Comment:
A new k8s bump is required upstream for 1.29.

Reply:
I think this file is Outdated. Upstream is using 1.29 and so as the PR

https://github.com/openshift/ansible-operator-plugins/pull/4/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R12